Assets/Scripts/MarchingCubesTunnelGenerator.cs

```diff
@@ -7,327 +7,325 @@
 
 using PathCreation;
```

Comment:
This is easier to review if you click the gear above and toggle "Hide whitespace" on. I removed an indent level from much of the file after removing the `namespace MarchingCubesProject`, which is from the Skrawk project.